jujugui/static/gui/src/app/views/utils.js

```diff
@@ -36,6 +36,38 @@ YUI.add('juju-view-utils', function(Y) {
     @class utils
   */
 
+
+  /**
+     Get a list of all the supported series.
+
+     @method getSeriesList
+     @return {Object} A collection of series.
+   */
+  utils.getSeriesList = function() {
+    // For a list of supported series in Juju see:
+    // https://github.com/juju/charmstore/blob/v5-unstable/internal/
+    // series/series.go#L37
+    return {
+      precise: {name: 'Precise Pangolin 12.04 LTS'},
```

Comment:
I think the list should be the same than here 
https://github.com/juju/charmstore/blob/v5-unstable/internal/series/series.go#L42
which also means one day we need to get that list from an endpoint in charmstore instead.